src/sage/groups/braid.py

```diff
@@ -2901,58 +2924,58 @@ def _LKB_matrix_(self, braid, variab):
             for i in braid[1:]:
                 A = A*self._LKB_matrix_((i,), variab)
             return A
-        l = list(Set(range(n)).subsets(2))
+        n2 = list(Set(range(n)).subsets(2))
```

Comment:
This is probably better to be replaced by `list(itertools.combinations(range(n), 2))`

Reply:
I made only style changes. I agree for the change but I prefer to keep as it is since for the sequel it is important for the elements of `n2` to be sets.

Reply:
Do you mean a Sage `Set` or is a Python `set` sufficient? It is easy enough to cast them as
```python
[set(X) for X in itertools.combinations(range(n), 2)]
```
and this should be faster (although it is less obvious to a non-Python programmer what the line is doing).

Reply:
Changed, but I get an error if I do not import it previously, so I added at the beginning `from itertools import combinations` and remove `itertools` at this place.

Reply:
Sorry, I should have been explicit about the import (I am used to always having to do it).